customTests/azure_ib_write_bw_gdr.nhc

```diff
@@ -62,7 +62,7 @@ function run_ib_bw_gdr(){
   dbg "ib_write_lb_${device}: $ib_bandwidth Gbps"
   if (( $(echo "$ib_bandwidth < $EXP_IB_BW" | bc -l) )); then
 	 log "$IB_WRITE_BW_OUT2"
-    die 1 "$FUNCNAME: $IB_WRITE_BW, IB=$device, $device_peer, IB BW (expected > $EXP_IB_BW Gbps, but measured $ib_bandwidth Gbps. FaultCode: NHC2003"
+    die 1 "check_ib_bw_gdr: $IB_WRITE_BW, IB=$device, $device_peer, IB BW (expected > $EXP_IB_BW Gbps, but measured $ib_bandwidth Gbps. FaultCode: NHC2003"
```

Comment:
I assume this is to make the die message consistent for this NHC test as this func is wrapped by `check_ib_bw_gdr`. Just want to make sure there is no parser that could have looked for `run_ib_bw_gdr`

Reply:
Yea this is more so to have the function name reported be the same as the check's name (which would be the initial function NHC enters)

Reply:
Addressed